common/libs/values.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import type Big from 'big.js';
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
I agree. @crptm wanted Big because it helped with flow types, but the fact that bignumber supports hex out of the box is too much to pass up.

Let's make that change!